Assets/Plugins/Source/Editor/PreProcessConfigConfirmation.cs

```diff
@@ -0,0 +1,70 @@
+﻿/*
+* Copyright (c) 2021 PlayEveryWare
+* 
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal
+* in the Software without restriction, including without limitation the rights
+* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+* copies of the Software, and to permit persons to whom the Software is
+* furnished to do so, subject to the following conditions:
+* 
+* The above copyright notice and this permission notice shall be included in all
+* copies or substantial portions of the Software.
+* 
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*/
+
+using UnityEditor.Build.Reporting;
+using UnityEditor.Build;
+using UnityEditor;
+using UnityEngine;
+using System.IO;
+using System.Collections.Generic;
+
+class PreProcessConfigConfirmation : IPreprocessBuildWithReport
+{
+    public int callbackOrder { get { return 0; } }
+    public void OnPreprocessBuild(BuildReport report)
+    {
+        Debug.Log("PreProcessConfigConfirmation.OnPreprocessBuild for target: " + report.summary.platform);
+        BuildTarget target = report.summary.platform;
+
+        Dictionary<BuildTarget, string> fileDictionary = new Dictionary<BuildTarget, string>();
+        fileDictionary.Add(BuildTarget.StandaloneWindows,   "Assets/StreamingAssets/EOS/EpicOnlineServicesConfig.json");
```

Comment:
nit: You can actually initialize a dictionary directly without adding key/value pairs after. No need to change it, but nice to know about C# initialization.
```
Dictionary<BuildTarget, string> fileDictionary = new Dictionary<BuildTarget, string> {
  { BuildTarget.StandaloneWindows, "Assets/StreamingAssets/EOS/EpicOnlineServicesConfig.json" },
  ...
};
```